hastory-cli/test/Hastory/Cli/OptParseSpec.hs

```diff
@@ -186,8 +186,21 @@ describeCommand =
                 "--storage-username=steven",
                 "--storage-password=letmein"
               ]
-            expectedSyncFlags = SyncFlags (Just url) (Just $ Username "steven") (Just "letmein")
+            expectedSyncFlags = RemoteStorageFlags (Just url) (Just $ Username "steven") (Just "letmein")
         cmd `shouldBe` CommandSync expectedSyncFlags
+    context "user provides the 'register' command"
+      $ it "parses to CommandRegister"
+      $ do
+        url <- parseBaseUrl "api.google.com"
+        let (Success (Arguments cmd _flags)) = runArgumentsParser args
+            args =
+              [ "register",
+                "--storage-server=api.google.com",
```

Comment:
Super nit: let's make this `hastory.cs-syd.eu`  instead of referring to google.

Reply:
No worries - makes complete sense to me.